src/game/models/mod.rs

```diff
@@ -2,157 +2,17 @@
 
 use std::f64;
 use std::f64::consts::PI;
-use std::ops::{Add, AddAssign, Rem, RemAssign, Sub, SubAssign, Div, DivAssign};
 
 use opengl_graphics::GlGraphics;
-use piston_window::{Context, UpdateArgs, Size};
-use rand;
+use piston_window::{Context, UpdateArgs};
 
 pub mod player;
```

Comment:
Minor, but I try to alphabetize lists in general to keep things easy to scan through and find. Let's do that with the `pub mod`'s here too.